.gitlab-ci.yml

```diff
@@ -2201,11 +2201,23 @@ pupernetes-master:
   - inv -e e2e-tests --image=datadog/agent-dev:master-py2
   - inv -e e2e-tests --image=datadog/agent-dev:master-py3
 
-pupernetes-tags:
+pupernetes-tags-6:
   <<: *pupernetes_template
   <<: *run_when_triggered_on_tag_6
+  variables:
+    PYTHON_RUNTIMES: '2,3'
```

Comment:
Is this variable needed in the e2e tests? Otherwise we can drop it.

Reply:
That's the same logic than what we do in `tag_release` targets so I'd say yes.